File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		int max = 0;
		int index = 0;
		for(int i = 0; i < 9; i++) {
			int n = Integer.parseInt(br.readLine());
			if(n > max) {
				max = n;
				index = i;
			}
		}
		
		System.out.println(max);
		System.out.println(index+1);
	}
}

File: 백준/Bronze/2562. 최댓값/최댓값.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    var array = Array(9,{0})

    for (i in 0 until array.size){
        val n = Integer.parseInt(br.readLine())
        array[i] = n
    }

    var max = 0
    var order = 0;
    for (i in 0 until array.size){
        if (array[i] > max){
            max = array[i]
            order = i+1
        }
    }

    println(max)
    println(order)

}